Assets/Scripts/ssjAug2018/World/Mailbox/MailboxManager.cs

```diff
@@ -13,10 +13,10 @@ public class MailboxManager : SingletonBehavior<MailboxManager>
         protected static readonly System.Random Random = new System.Random();
 
         // TODO: Fix this. Everything's coming up null when I try to find the player.
-        private Player _player;
         private PlayerData _playerData = PlayerManager.Instance.PlayerData;
```

Comment:
Don't hold a reference to the data here, just access it directly when you need it